p4-16/psa/PSA.mdk

```diff
@@ -675,16 +675,17 @@ follow.
     if (ostd.clone) {
         if (ostd.clone_session_id value is supported) {
             cos = class_of_service configured for ostd.clone_session_id in PRE
-            ep = egress_port configured for ostd.clone_session_id in PRE
+	    egress_port = egress_port in the set of pairs of egress_port and instance configured for ostd.clone_session_id in PRE
+	    instance = instance in the set of pairs of egress_port and instance configured for ostd.clone_session_id in PRE
```

Comment:
Since this is pseudo code, would it be too outside P4 to write:
`(egress_port, instance) = egress_port and instance in the set of configured sessions for ostd.clone_session_id in PRE`?

Reply:
hmm, just as you did in line 1023 :)